app/assets/javascripts/admin/components/theme-setting-editor.js.es6

```diff
@@ -1,12 +1,17 @@
 import BufferedContent from "discourse/mixins/buffered-content";
 import SettingComponent from "admin/mixins/setting-component";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
 
 export default Ember.Component.extend(BufferedContent, SettingComponent, {
   layoutName: "admin/templates/components/site-setting",
   _save() {
-    return this.model.saveSettings(
-      this.get("setting.setting"),
-      this.get("buffered.value")
-    );
+    return ajax(`/admin/themes/${this.model.id}/setting`, {
+      type: "POST",
```

Comment:
In Rails the convention is to use `PUT` for an update and `POST` for a create. In this case it seems more like an update than a create so I think it should use `PUT`.